coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/config/SecondaryConfig.scala

```diff
@@ -5,12 +5,12 @@ import com.typesafe.config.Config
 import java.io.File
 
 trait StoreConfig {
-  val storeCapacityMB: Int
+  val storeCapacityMB: Long
   val acceptingNewDatasets: Boolean
 }
 
 class StoreConfigCfg(config: Config, root: String) extends ConfigClass(config, root) with StoreConfig {
-  val storeCapacityMB = getInt("storeCapacityMB")
+  val storeCapacityMB = getInt("storeCapacityMB").toLong
```

Comment:
Could this just be `getLong` or does that method not exist?

Reply:
That method doesn't exist yet.  I open another PR to provide that in the library.  But for hotfix-ish purpose.  I rather do this as a baby step.